examples/ola-streaming-client.cpp

```diff
@@ -78,11 +84,30 @@ int main(int argc, char *argv[]) {
 
   if (FLAGS_dmx.str().empty()) {
     string input;
+    bool have_universe = false;
+    unsigned int read_universe = FLAGS_universe;
+
     while (!terminate && std::cin >> input) {
-      ola::StringTrim(&input);
-      SendDataFromString(&ola_client, FLAGS_universe, input);
+      if (!have_universe && FLAGS_universe_from_source) {
+        if (!ola::StringToInt(input, &read_universe, true)) {
+          OLA_FATAL << "Could not convert universe number.";
+          exit(1);
+        }
+
+        have_universe = true;
+        continue;
+      }
+
+      if (have_universe || !FLAGS_universe_from_source) {
+        SendDataFromString(&ola_client, read_universe, input);
+        have_universe = false;
+      }
     }
   } else {
+    if (FLAGS_universe_from_source) {
+      OLA_FATAL << "Not reading from STDIN. Use -u to specify universe.";
```

Comment:
This should probably be a cout message in case my logging is going to syslog for example.

Reply:
Hm, but I'd argue that if you're logging to syslog then you'd expect it to show up there, no?

Reply:
We generally differentiate between messages to the user and logging. It's a bit of grey area, but it's going to bomb out so you probably want to see why immediately. The same as the usage stuff.

Reply:
Either way, there's a cout in SendDataFromString which is inconsistent!

Reply:
Sorry, I've just realised this only happens when it's NOT reading from STDIN, so could be running in a script sent to syslog.